lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
That probably makes the most sense. I'm slightly nervous that now we have to train translators to not include ``{`` in their strings, versus ``${`` which is very unlikely to show up accidentally.